app/helpers/account/profiles_helper.rb

```diff
@@ -0,0 +1,9 @@
+module Account::ProfilesHelper
+  def profiles_avatar(user)
+    if user.with_avatar?
+      image_tag url_for(user.avatar), class: 'is-rounded'
```

Comment:
the difference in if...else statement is only in url, please move it in variable and use image tag without condition